From fd8c0d96d88ade379e6de9f618c4eb7b2e20f561 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Mon, 26 Jul 2021 11:15:10 +0200
Subject: [PATCH] Revert "netifd: update to the latest version"

I am going to revert this as it seems there are some issues with netifd
with the recent update. It does not give any addresses and my computer
does not want to get the lease from router.

Jul 26 09:15:05 turris odhcpd[2248]: A default route is present but there is no public prefix on guest_turris thus we don't announce a default route!
Jul 26 09:15:05 turris odhcpd[2248]: Failed to send to ff02::1%guest_turris@br-guest_turris (Address not available)
Jul 26 09:15:05 turris odhcpd[2248]: Failed to send to ff02::1%lan@br-lan (Address not available)
Jul 26 09:15:21 turris odhcpd[2248]: A default route is present but there is no public prefix on guest_turris thus we don't announce a default route!
Jul 26 09:15:21 turris odhcpd[2248]: Failed to send to ff02::1%guest_turris@br-guest_turris (Address not available)
Jul 26 09:15:21 turris odhcpd[2248]: Failed to send to ff02::1%lan@br-lan (Address not available)
Jul 26 09:15:37 turris odhcpd[2248]: A default route is present but there is no public prefix on guest_turris thus we don't announce a default route!
Jul 26 09:15:37 turris odhcpd[2248]: Failed to send to ff02::1%guest_turris@br-guest_turris (Address not available)
Jul 26 09:15:37 turris odhcpd[2248]: Failed to send to ff02::1%lan@br-lan (Address not available)

This was also mentioned on #openwrt-devel.
Reference 26/07/2021:
09:18:54	<ynezz>	FYI mvebu/omnia doesn't boot with latest master and 21.02
09:19:18	<ynezz>	60fad8f82b was last working commit on omnia and 21.02
09:20:17	<ynezz>	90db922f88 was last working commit on omina and master
09:20:40	<ynezz>	https://gitlab.com/ynezz/openwrt-device-runtime-testing/-/pipelines/342708318/failures
09:22:04	<ynezz>	tests fails during device bringup phase, when the controller waits dor device under test LAN network availability

This reverts commit 089efd61e9a6cdc0ea39c184d37bc8ebbe03175c.
---
 package/network/config/netifd/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/package/network/config/netifd/Makefile b/package/network/config/netifd/Makefile
index 35035519f9..f80f07a4ff 100644
--- a/package/network/config/netifd/Makefile
+++ b/package/network/config/netifd/Makefile
@@ -5,9 +5,9 @@ PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL=$(PROJECT_GIT)/project/netifd.git
-PKG_SOURCE_DATE:=2021-07-23
-PKG_SOURCE_VERSION:=17e453bd68b41780b6564dafa8358efc29a00930
-PKG_MIRROR_HASH:=06b6e10a71e3002fa03579846b97af29be9503d61486486c639dbfae423b398f
+PKG_SOURCE_DATE:=2021-07-14
+PKG_SOURCE_VERSION:=7f24a063475e1e2be4e0c516a5b62c3fae5ec542
+PKG_MIRROR_HASH:=3fad25b8655e01e20f54b6c6decad6a660a34ab3ca2de97fe3c9b7db5aa485fe
 PKG_MAINTAINER:=Felix Fietkau <nbd@nbd.name>
 
 PKG_LICENSE:=GPL-2.0
-- 
2.30.2

